spec/controllers/api/v1/app_identifier_visitor_configs_controller_spec.rb

```diff
@@ -2,14 +2,14 @@
 
 RSpec.describe Api::V1::AppIdentifierVisitorConfigsController do
   describe "#show" do
-    let(:app) { FactoryBot.create(:app) }
-    let(:identifier_type) { FactoryBot.create(:identifier_type, name: "clown_id") }
-    let(:feature_gate) { FactoryBot.create(:feature_gate, name: "blab_enabled", registry: { false: 50, true: 50 }) }
-    let(:visitor) { FactoryBot.create(:visitor) }
-    let(:identifier) { FactoryBot.create(:identifier, visitor: visitor) }
+    let(:app) { create(:app) }
+    let(:identifier_type) { create(:identifier_type, name: "clown_id") }
+    let(:feature_gate) { create(:feature_gate, name: "blab_enabled", registry: { false: 50, true: 50 }) }
+    let(:visitor) { create(:visitor) }
+    let(:identifier) { create(:identifier, visitor: visitor) }
 
     it "has knocked-out weightings and it doesn't include a non-force assignment for a feature gate that isn't feature complete" do
-      FactoryBot.create(
+      create(
```

Comment:
I think we typically don't mix in / rely on FactoryBot's spec helper mixin -- instead going with the explicit `FactoryBot.create(...)` calls (etc).

Thoughts? Was this a linter autocorrect that we can revert?

Reply:
This was an automated linter fix. I can revert the fix and turn off this rule if that's our standard.